global/scripts/select_dwi_vols

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+if [ "$4" == "" ];then
+    echo ""
+    echo "Usage: select_dwi_vols <data> <bvals> <output> <approx_bval> [other options]"
+    echo ""
+    echo " Optional arguments:"
+    echo ""
+    echo " -b <value>    additional bvalues to be extracted (can be used multiple times)"
+    echo " -m            output mean instead of concat"
+    echo " -v            output variance instead of concat"
+    echo " -obv <bvecs>  produce <output>.bval and <output>.bvec"
+    echo " -db <delta_bvals> only include volumes with offset in b-value less than this value (default: 100 s/mm2)"
```

Comment:
Unclear from this description, does the offset only apply for frames less than the `-b` value, or does it apply in either direction?

Reply:
The absolute value of the offset should be smaller than this number. I've clarified this.